dspace/config/clarin-dspace.cfg

```diff
@@ -246,3 +246,7 @@ file.preview.enabled = false
 ### Storage service ###
 # Synchronization is NOT enabled by default
 sync.storage.service.enabled = true
+
+
+### The build version is stored in the specific file ###
+build.version.file.path =
```

Comment:
fill this with a link ponting to a file in dspace/config directory named `VERSION_D`